packages/trader/src/App/Components/Elements/DataTable/data-table.jsx

```diff
@@ -51,37 +50,12 @@ class DataTable extends React.PureComponent {
         };
     }
 
```

Comment:
So now that we are leveraging flex & 100% width, we don't need to listen for resize events anymore?

Reply:
resizeDimensions was causing some issues in compare accounts. I think its better to leave the resizing to the browser by using 100%.